### DISCUSSION TITLE
A small update is required in the proof in Box 3.1 at page 42

### DISCUSSION THREAD

**Discussion by eatam23** [MathFoundationRL/Book-Mathematical-Foundation-of-Reinforcement-Learning#11] [General]
In the the proof given in Box 3.1 at page 42, it is said that "A sequence $x_1, x_2,\cdots, \in \mathbb{R}$ is called  Cauchy..."  However, the correct thing should be $x_1, x_2,\cdots, \in \mathbb{R}^d$ since $x$ is a vector.

**Comment by MathFoundationRL** [comment]
Noted and thanks!
